Gems/PhysX/Code/Tests/EditorColliderComponentTests.cpp

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <Tests/PhysXColliderComponentmodeTests.h>
+#include <Source/EditorRigidBodyComponent.h>
+
+namespace UnitTest
+{
+    class ColliderPickingFixture : public PhysXEditorColliderComponentManipulatorFixture
+    {
+    public:
+        void SetUpEditorFixtureImpl() override;
+        //! Clicks at a given position and returns the entities that are selected.
+        AzToolsFramework::EntityIdList ClickAndGetSelectedEntities(AzFramework::ScreenPoint screenPoint);
+
+        const float m_uniformScale = 1.0f;
+        const AZ::Quaternion m_shapeRotation = AZ::Quaternion(0.0f, 0, 0, 1);
+        const AZ::Quaternion m_entityRotation = AZ::Quaternion(0, 0, 0, 1);
+        const AZ::Vector3 m_shapeOffset = AZ::Vector3(0, 0, 0);
+        const AZ::Vector3 m_entityTranslation = AZ::Vector3(5.0f, 15.0f, 10.0f);
```

Comment:
Are these needed to be public?

It's also odd that they are all `const`, why is that? Do they change per instance or can they be static?